lib/Sema/TypeCheckStmt.cpp

```diff
@@ -2145,7 +2069,6 @@ TypeCheckFunctionBodyUntilRequest::evaluate(Evaluator &evaluator,
 
   // FIXME(InterfaceTypeRequest): Remove this.
   (void)AFD->getInterfaceType();
```

Comment:
You should be able to get rid of this too.

Reply:
Tried it, there are a couple of code completion issues, likely a missing forced interface type. Leaving for a future PR.